docker-compose.yml

```diff
@@ -17,43 +16,31 @@ services:
       - P2P_ENVIRONMENT=${P2P_ENVIRONMENT}
       - P2P_DISCOVERY_NAMESPACE=${P2P_DISCOVERY_NAMESPACE}
     ports:
-      - "8080:8080"
-    networks:
-      - app-network
```

Comment:
If there is only one network it's not necessary to explicitly define one. Compose will create a default one.